cupy/indexing/insert.py

```diff
@@ -66,7 +67,77 @@ def put(a, ind, v, mode='wrap'):
     a.put(ind, v, mode=mode)
 
 
-# TODO(okuta): Implement putmask
+_putmask_kernel = core._kernel.ElementwiseKernel(
```

Comment:
Would you remove a private submodule name?
```suggestion
_putmask_kernel = core.ElementwiseKernel(
```